core/src/main/resources/hudson/model/Job/buildTimeTrend.jelly

```diff
@@ -31,6 +31,14 @@ THE SOFTWARE.
     <l:breadcrumb title="${%Build Time Trend}" />
     <st:adjunct includes="hudson.model.Job.buildTimeTrend_resources" />
     <l:main-panel>
+      <template id="icons">
```

Comment:
That id seems to be rather generic and bears the risk of collision. I would suggest to apply here the rules as defined in the design-library and prefix with `jenkins-` or `app-`

Reply:
Done.